### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jonathoncove2**
## Description

Added a brief example on adding messages to a notification 

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ x] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)
v10

## Deadline (if relevant)

N/A

### CODE REVIEW

File: 10/umbraco-cms/reference/notifications/README.md [hunk 1]
```diff
@@ -275,6 +275,17 @@ Useful for manipulating the model before it is sent to an editor in the backoffi
 
 Umbraco uses notifications to allow people to hook into different workflow processes. This notification pattern is extensible, allowing you to create and publish custom notifications, and other people to observe and hook into your custom processes. This approach can be useful when creating Umbraco packages. For more information on how you create and publish your own notifications, see the [creating and publishing notifications](creating-and-publishing-notifications.md) article.
 
+##Showing messages in the CMS
```
Comment (sofietoft) [inline 10/umbraco-cms/reference/notifications/README.md:278]:
```suggestion
## Showing messages in the CMS
```

File: 10/umbraco-cms/reference/notifications/README.md [hunk 2]
```diff
@@ -275,6 +275,17 @@ Useful for manipulating the model before it is sent to an editor in the backoffi
...
+
+You can inform the Umbraco user of the status of your notification by adding messages to the ```notification.Messages``` property.
```
Comment (sofietoft) [inline 10/umbraco-cms/reference/notifications/README.md:280]:
```suggestion
You can inform the Umbraco user of the status of your notification by adding messages to the `notification.Messages` property. 
```

File: 10/umbraco-cms/reference/notifications/README.md [hunk 3]
```diff
@@ -275,6 +275,17 @@ Useful for manipulating the model before it is sent to an editor in the backoffi
 
 Umbraco uses notifications to allow people to hook into different workflow processes. This notification pattern is extensible, allowing you to create and publish custom notifications, and other people to observe and hook into your custom processes. This approach can be useful when creating Umbraco packages. For more information on how you create and publish your own notifications, see the [creating and publishing notifications](creating-and-publishing-notifications.md) article.
 
+##Showing messages in the CMS
+
+You can inform the Umbraco user of the status of your notification by adding messages to the ```notification.Messages``` property. 
+
+For example:
```
Comment (sofietoft) [inline 10/umbraco-cms/reference/notifications/README.md:282]:
```suggestion
For example:

```

**Review by sofietoft — COMMENTED** [review]
Thanks for the PR @jonathoncove2 !

I've made some tiny suggestions. Let me know if something doesn't look right.

**Comment by jonathoncove2** [comment]
Thanks for the tweaks!

**Comment by sofietoft** [comment]
Thanks for considering the suggestions @jonathoncove2 ! 💪 

Would it make sense to add a little more context to this code sample?
Like, in which file would this code go, and when would you normally use this kind of notification? 🤔

**Comment by sofietoft** [comment]
Hi @jonathoncove2 !

Did you see my previous message? 😄

**Comment by jonathoncove2** [comment]
Apologies @sofietoft , I've made some edits now which give some more detail. Likely needs umbraco-ifying  a bit

File: 10/umbraco-cms/reference/notifications/README.md [hunk 4]
```diff
@@ -275,6 +275,36 @@ Useful for manipulating the model before it is sent to an editor in the backoffi
 
 Umbraco uses notifications to allow people to hook into different workflow processes. This notification pattern is extensible, allowing you to create and publish custom notifications, and other people to observe and hook into your custom processes. This approach can be useful when creating Umbraco packages. For more information on how you create and publish your own notifications, see the [creating and publishing notifications](creating-and-publishing-notifications.md) article.
 
+## Showing messages in the CMS
+
+When handling notifications for CMS actions, you can inform the Umbraco user of the status of your notification by adding to the `notification.Messages` property within the `Handle` function.
```
Comment (sofietoft) [inline 10/umbraco-cms/reference/notifications/README.md:280]:
```suggestion
When handling notifications for CMS actions, you can inform the Umbraco user of the status of your notification. This is done by adding to the `notification.Messages` property within the `Handle` function. 
```

**Review by sofietoft — COMMENTED** [review]
This is great @jonathoncove2 ! 😄 Thanks.

I've made a suggestion - should shorten the sentence, but let me know if you think it still makes sense!

File: 10/umbraco-cms/reference/notifications/README.md (hunk 4 above)
Comment (jonathoncove2) [reply 10/umbraco-cms/reference/notifications/README.md:280]: That looks good to me! It reads much better that way

**Comment by sofietoft** [comment]
Thanks for considering the suggestion @jonathoncove2 ! 😄 

Looks like we're ready to merge this one.
I'll do so, once the checks complete.

**Comment by nul800sebastiaan** [comment]
@jonathoncove2 Good day! You still have a few days to let us know what you want for Hacktoberfest 🎁, we'd love it if it was an active choice!
If you just want us to plant trees 🌳 then let me know or give the form a quick fill (only 2 fields to fill in for trees).
https://feedback.umbraco.com/umbraco-hacktoberfest-2024/

Thanks!

**Pull request merged**